lua/telescope/pickers.lua

```diff
@@ -491,6 +491,60 @@ function Picker:hide_preview()
   -- 2. Resize prompt & results windows accordingly
 end
 
+-- TODO: update multi-select with the correct tag name when available
+--- A simple interface to remove an entry from the results window without
+--- closing telescope. This either deletes the current selection or all the
+--- selections made using multi-select. It can be used to define actions
+--- such as deleting buffers or files.
+---
+--- Example usage: >
```

Comment:
Do code blocks work? In vim help it is denoted with enclosing angle brackets: `> ... <`

Reply:
You can use `<pre>` tags for code blocks

Reply:
Ah, no what I mean is does it highlight in the `help` buffer. [This previewers code block](https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/previewers/init.lua#L75) does not highlight.

Reply:
yeah, apparently they don't..

Reply:
`<pre>` only disables the markdown formatter.

:thinking: Maybe i should add that to `docgen` :thinking:

Reply:
Yeah, that would be good. The highlights will help separate out the code, maybe `<code>`?